server/endpoints/Review.ts

```diff
@@ -0,0 +1,54 @@
+import { body } from "express-validator";
+import { getCrossListOR } from "common/CourseCard";
+import { Endpoint } from "../endpoints";
+import { Reviews, Classes } from "../dbDefs";
+
+interface CourseId {
+  courseId: string;
+}
+
+export const getCourseById: Endpoint<CourseId> = {
+  guard: [body("courseId").notEmpty().isAscii()],
+  callback: async (courseId: CourseId) => {
+    try {
+      // check: make sure course id is valid and non-malicious
+      const regex = new RegExp(/^(?=.*[A-Z0-9])/i);
+      if (regex.test(courseId.courseId)) {
+        return await Classes.findOne({ _id: courseId.courseId }).exec();
+      }
+      return null;
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log("Error: at 'getCourseById' method");
+      // eslint-disable-next-line no-console
+      console.log(error);
+      return null;
+    }
+  },
+};
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
It would be nice to have a little bit of documentation here. Just a couple of words should do, I think.